File: app/src/main/java/mariavv/fitnesspal/data/db/Meal.java
package mariavv.fitnesspal.data.db;

public enum Meal {
    BREAKFAST,
    LAUNCH,
    DINNER;

    public String getValue() {
        switch (this) {
            case BREAKFAST:
                return "Завтрак";
            case LAUNCH:
                return "Обед";
            case DINNER:
                return "Ужин";
        }
        return "";
    }
}

File: app/src/main/java/mariavv/fitnesspal/data/db/Meal.kt
package mariavv.fitnesspal.data.db

enum class Meal {
    BREAKFAST,
    LAUNCH,
    DINNER;

    val value: String
        get() {
            return when (this) {
                BREAKFAST -> "Завтрак"
                LAUNCH -> "Обед"
                DINNER -> "Ужин"
            }
        }
}
